### DISCUSSION TITLE
Acceso a organizacion LaunchX sera siempre?

### DISCUSSION THREAD

**Discussion by atzsamuel** [Launch-X-Latam/MisionFrontEnd#20] [Q&A]
Tendremos acceso siempre a la organizacion github de LaunchX o al finalizar nos sacaran ?

**Comment by JosafatJimenezB** [comment]
No, el bot para las invitaciones solo se activa un tiempo así que ingresa a la organización cuando puedas, saludos!

**Comment by Emilio17MC** [comment]
Me parece que el acceso es permanente, pues es una forma de demostrar que participaste en este programa de formación.

**Comment by rfponce** [comment]
Tengo entendido que además de aprender, la idea es crear una comunidad permanente. Así que lo más probable es que no nos saquen.

**Comment by doguedogue** [comment]
Forever and ever 🚀
